_PR.md

```diff
@@ -0,0 +1,58 @@
+# Changes
+
+This is just the file I'm using to keep track of everything. It won't be part of the final PR.
+
+## TODO
+
+- update all examples
+- update examples for yew-router (fix #1377)
+- update examples for yewtil
+
+## Somewhat related issues
+
+- #1073 (should probably be closed now)
+- #1016 (promises some good stylin' for the examples but it's been a while)
+
+## Examples
+
+## 1. counter
+
+## 2. crm
+
+Modified to make the example more modular.
+What used to be a single component handling a lot of message spaghetti is now split into dedicated components.
+
+## 3. custom_components
+
+**Should be updated or removed**.
+
+## 4. dashboard
+
+## 5. file_upload
+
+## 5. fragments
+
+**Should be updated or removed**.
+
+## 6. futures
+
+Renamed from "futures_wp".
+
+## 7. game_of_life
+
+Split code across two files.
+
+## 8. inner_html
+
+## 16. multithread
+
+**TODO** this example is special!
```

Comment:
yeah I wonder if this will need extra support in `trunk`. It's really overdue that we get this working very easily given Yew's promise of easily setting up multi-threaded apps with web workers

Reply:
@jstarry what sort of additional setup / support would be needed? Definitely a lot to be done yet in Trunk, but I'm happy to prioritize certain issues if it would help.

Reply:
Disregard, sounds like @siku2 is going to pop an issue in Trunk for this tomorrow. All good.

Reply:
@jstarry, I opened an issue for this: thedodd/trunk#46. Can you think of any setup that wouldn't be covered by this?

Reply:
Nope I think that covers it, very exciting!